core/metacling/src/TCling.cxx

```diff
@@ -4102,7 +4102,7 @@ TCling::CheckClassInfo(const char *name, Bool_t autoload, Bool_t isClassOrNamesp
                int lineNumber = 0;
                bool success = false;
                std::tie(success, lineNumber) =
-                  ROOT::TMetaUtils::GetTrivialIntegralReturnValue(implLineFunc.GetMethodDecl(), *fInterpreter);
+                  ROOT::TMetaUtils::GetTrivialIntegralReturnValue(implLineFunc.GetAsFunctionDecl(), *fInterpreter);
```

Comment:
Could GetTrivialIntegralReturnValue return a nullptr (since it has a dyn_cast)?

Reply:
I don't understand your question. I'd understand if it were
> Could GetAsFunctionDecl return a nullptr (since it has a dyn_cast)?

`implLineFunc.IsValid()` is checked before; we do not support using decls for the `ImplLine` function, so by definition it must be a `FunctionDecl`.

Reply:
fair enough.